src/cosmos/mint/v1beta1/tx.registry.ts

```diff
@@ -0,0 +1,54 @@
+//@ts-nocheck
```

Comment:
do we really need mint, nft etc. modules generated code? why did they appear? can we rm them if we don't need them?

Reply:
They are because cosmos has it, i think. Not sure how to remove these automatically for now